src/classes/RD2_VisualizeScheduleControllerTest.cls-meta.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>47.0</apiVersion>
```

Comment:
Should be api v46

Reply:
Interesting.  I thought our approach was to always go with latest.  What drives our choice?